.env.local.example

```diff
@@ -1,2 +1,10 @@
 NEXT_PUBLIC_GOOGLE_MAPS_API_KEY=CREATE_YOUR_OWN_KEY
-NEXT_PUBLIC_GOOGLE_MAPS_MAP_ID=NUMBERS_AND_STUFF
\ No newline at end of file
+NEXT_PUBLIC_GOOGLE_MAPS_MAP_ID=NUMBERS_AND_STUFF
+
+# Google Cloud stuff for backend functionality
```

Comment:
Make sure to update your .env.local with these! I can help you set up the Base64 key later.